sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ExecutionStateSampler.java

```diff
@@ -187,13 +253,37 @@ public class ExecutionStateTracker implements BundleProgressReporter {
     // Read and written by the ExecutionStateSampler thread
     private long transitionsAtLastSample;
 
+    // Ignore the @UnderInitialization for metricsContainer since both this and it will be
```

Comment:
With new change I think there it will not throw @UnderInitialization and we can remove `@SuppressWarnings({"assignment", "argument"})`

Reply:
It still will bring this up since the thing under initialization is the execution state tracker. Fixed up comment.

Reply:
ACK